internal/sirius/cases_by_assignee.go

```diff
@@ -14,6 +15,13 @@ type Case struct {
 	ReceiptDate SiriusDate `json:"receiptDate"`
 	Status      string     `json:"status"`
 	TaskCount   int        `json:"taskCount"`
+	WorkedDate  SiriusDate `json:"workedDate"`
+}
+
+func (c Case) IsWorked() bool {
+	day := 24 * 60 * time.Minute
+
+	return c.WorkedDate.Truncate(day).Equal(time.Now().Truncate(day))
```

Comment:
If it doesn't count as worked after 24 hours, I think we should build this into `Lpa::getWorkedDate` in the API rather than leaving it up to downstream services to decide. Otherwise we have to implement this rule repeatedly.

Reply:
Good point, should just expose `getWorked()` on CaseItem with the logic in